lib/index.coffee

```diff
@@ -51,6 +53,29 @@ streamToString = (stream) ->
 		.on 'end', ->
 			resolve(Buffer.concat(chunks).toString())
 
+setupMetrics = (metrics) ->
+	metrics.describe('gc_space_reclaimed', 'Disk space the GC must free')
```

Comment:
this should be `metrics.describe.counter`

Reply:
:+1:

Reply:
Counters need not increase _monotonically_, but they cannot decrease (with the exception of relatively "rare" things like total resets). If this process is not daemonized, how is this data being exposed for scraping?

Reply:
The builder has a singleton of this GC object which exposes a function to to the cleanup. At construction time, the GC describe the metrics and updates them every the builder requests a garbage collection.

The `gc_space_reclaimed` metric can indeed decrease from call to call, since it depends on the state of the system at that moment.

Reply:
So, the difference in this case between a counter and a gauge would be, if we use a gauge, we are merely *setting* the value each time the GC runs, and the scrape by prometheus will pick up whatever value was last set (and, if we *don't* run a GC before the next scrape, it will pick up the same value, leading to a confusion, is this the same number for a new GC run, or the stale value from the last GC, unchanged?)

I think we should use a counter here such that `increase(gc_space_reclaimed[$interval])` will give us the total space reclaimed in a given time interval, decoupling us somewhat from questions of how the two frequencies of `GC-run-frequency` and `scrape-frequency` interact.

Reply:
Awesome, makes complete sense :)